bounswe/bounswe2017group5
# Post creation with data templates

## About

* **Related Pull Requests:** #139

Implement the frontend and backend for post creation with data templates.

## Visuals

![create-post-with-template](https://user-images.githubusercontent.com/13895224/33334798-2f060f40-d47c-11e7-9023-c94596c1f8a1.gif)
_Note:_ We show the recent posts at the top now, visual is a little bit behind :)

## Testing

- Create a data template or select an existing one.
- Fill in the data fields and click the submit button.
- The page should be reloaded and you should be able to see your new post at the top.
